rsocket-micrometer/src/main/java/io/rsocket/micrometer/MicrometerDuplexConnection.java

```diff
@@ -0,0 +1,244 @@
+      this.error = counter(connectionType, meterRegistry, ERROR, tags);
+      this.extension = counter(connectionType, meterRegistry, EXT, tags);
+      this.keepalive = counter(connectionType, meterRegistry, KEEPALIVE, tags);
+      this.lease = counter(connectionType, meterRegistry, LEASE, tags);
+      this.metadataPush = counter(connectionType, meterRegistry, METADATA_PUSH, tags);
+      this.next = counter(connectionType, meterRegistry, NEXT, tags);
+      this.nextComplete = counter(connectionType, meterRegistry, NEXT_COMPLETE, tags);
+      this.requestChannel = counter(connectionType, meterRegistry, REQUEST_CHANNEL, tags);
+      this.requestFireAndForget = counter(connectionType, meterRegistry, REQUEST_FNF, tags);
+      this.requestN = counter(connectionType, meterRegistry, REQUEST_N, tags);
+      this.requestResponse = counter(connectionType, meterRegistry, REQUEST_RESPONSE, tags);
+      this.requestStream = counter(connectionType, meterRegistry, REQUEST_STREAM, tags);
+      this.resume = counter(connectionType, meterRegistry, RESUME, tags);
+      this.resumeOk = counter(connectionType, meterRegistry, RESUME_OK, tags);
+      this.setup = counter(connectionType, meterRegistry, SETUP, tags);
+    }
+
+    @Override
+    public void accept(Frame frame) {
+      FrameType frameType = frame.getType();
+
+      switch (frameType) {
+        case RESERVED:
+          break;
+        case SETUP:
+          this.setup.increment();
+          break;
+        case LEASE:
+          this.lease.increment();
+          break;
+        case KEEPALIVE:
+          this.keepalive.increment();
+          break;
+        case REQUEST_RESPONSE:
+          this.requestResponse.increment();
+          break;
+        case REQUEST_FNF:
+          this.requestFireAndForget.increment();
+          break;
+        case REQUEST_STREAM:
+          this.requestStream.increment();
+          break;
+        case REQUEST_CHANNEL:
+          this.requestChannel.increment();
+          break;
+        case REQUEST_N:
+          this.requestN.increment();
+          break;
+        case CANCEL:
+          this.cancel.increment();
+          break;
+        case PAYLOAD:
+          break;
+        case ERROR:
+          this.error.increment();
+          break;
+        case METADATA_PUSH:
+          this.metadataPush.increment();
+          break;
+        case RESUME:
+          this.resume.increment();
+          break;
+        case RESUME_OK:
+          this.resumeOk.increment();
+          break;
+        case NEXT:
+          this.next.increment();
+          break;
+        case COMPLETE:
+          this.complete.increment();
+          break;
+        case NEXT_COMPLETE:
+          this.nextComplete.increment();
+          break;
+        case EXT:
+          this.extension.increment();
+          break;
+        default:
+          this.logger.warn("Skipping count of unknown frame type: {}", frameType);
```

Comment:
I'd suggest this be at debug level and still cause a counter to be incremented.  It has always bit me in server development to decide to be verbose about any error that a misconfigured or newer client can externally trigger.  Suddenly server performance is effected by excessive logging, while instead tracking a counter and then temporarily increasing logging in response to an alert is much safer IMHO.

Reply:
Reasonable enough.  Going with an `UNKNOWN` frame type.